demos/appyx-navigation/ios/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+plugins {
+    id("com.bumble.appyx.multiplatform")
+    kotlin("multiplatform")
+    kotlin("native.cocoapods")
+    id("org.jetbrains.compose")
+    id("com.google.devtools.ksp")
+}
+
+kotlin {
+
+    iosX64()
+    iosArm64()
+    iosSimulatorArm64()
+
+    cocoapods {
+        version = "1.0.0"
+        summary = "appyx-nvaigation ios module"
+        homepage = "https://bumble-tech.github.io/appyx/interactions/"
+        ios.deploymentTarget = "16.4"
+        podfile = project.file("../iosApp/Podfile")
+        framework {
+            baseName = "ios"
+            isStatic = true
+        }
+    }
+
+    sourceSets {
+        val iosX64Main by getting
+        val iosArm64Main by getting
+        val iosSimulatorArm64Main by getting
+        val iosMain by creating {
+            iosX64Main.dependsOn(this)
+            iosArm64Main.dependsOn(this)
+            iosSimulatorArm64Main.dependsOn(this)
+            dependencies {
+                implementation(project(":demos:common"))
+                implementation(project(":demos:appyx-navigation:common"))
+                implementation(project(":appyx-interactions:appyx-interactions"))
```

Comment:
I'm not sure we actually need this here, I think navigation dependency should be enough.

Reply:
Only using `:demos:appyx-navigation` now